### ISSUE TITLE
Looking for examples of vertical profiles with circular arc and clothoid

### ISSUE THREAD

**Issue by RickBrice** [buildingSMART/IFC4.x-IF#161]
Throughout my entire career I've only dealt with alignment vertical profiles having parabolic curves. I'm looking for some real world examples of vertical profiles with circular and clothoid curves. Sample plan sheets would be great. Can anyone help?

**Comment by evandroAlfieri** [comment]
I know for sure that in Italy, railway design, by regulation can only use circular arcs for the vertical profiles. Road design can use parabolas. [This test](https://github.com/buildingSMART/IFC4.x-IF/tree/main/tests/STN01) uses an Italian dataset, you'll find circular arcs in the vertical profile

**Comment by apinzenoehler** [comment]
Dear Richard, dear all!

As Evandro has already said, it depends on the relevant regulations.

Rail regulations for "track geometry design" focus on vehicle speed and safety. The necessary mathematical formulas are all somehow based on or connected to curvature.

Mathematical treatment of curvature is very easy for straight lines and circular arcs. That is probably the reason that in all rail regulations I have seen so far vertical smoothing for gradient changes is expressed using circular arcs.

The inclusion of clothoids into the vertical layout was introduced by one of the contributing IfcRail Stakeholders quite late during the requirements detection phase. Here again it is the regulation. Clothoids are applied for the design of high speed lines for travelspeeds of up to 400-500 km per hour. At such speeds this specific railway company uses vertical clothoids for better handling of gradient changes. "Better handling" means that the trains do not get airborne ;-)

Please note that in rail design the typical vertical smoothing circle combines a large value for the radius (more than 1000 meter, quite often something in the range of 20000 to 40000 meters) with very short circle arc length values (16 to 40 meter).

Therefore the circular arc is sometimes approximated by a parabolic curve to calculate values for setting out and acceptance of work purposes.

In a nutshell: 

- In rail design vertical smoothing of gradient changes is designed and documented using vertical circular arcs. 
- The vertical clothoid is used for extreme high speed design. 
- Parabolic curves are occasionally used as approximations for circular arcs especially if working in the field.

BR Andreas

**Comment by RickBrice** [comment]
Thank you. This helps. Is the application of vertical clothoids similar to horizontal? By that I mean, for horizontal it is usually tangent-clothoid-circular arc-clothoid-tangent, so for vertical is it typically grade-clothoid-circular arc-clothoid-grade? It probably doesn’t make sense to have grade-clothoid-grade because of the abrupt change in curvature.
